cohorts/cohort.py

```diff
@@ -643,8 +633,9 @@ def _load_single_patient_effects(self, patient, only_nonsynonymous, filter_fn):
         # Always take the top priority effect per variant so we end up with a single
         # effect per variant.
         nonsynonymous_effects = EffectCollection(
-            effects.drop_silent_and_noncoding().top_priority_effect_per_variant().values())
-        effects = EffectCollection(effects.top_priority_effect_per_variant().values())
+            list(effects.drop_silent_and_noncoding().top_priority_effect_per_variant().values()))
```

Comment:
`.values()` is an `odict_values` in python3 which can't be serialized. Another fix here might have been `itervalues()` instead.